app/server/src/controllers/indexController.ts

```diff
@@ -109,5 +107,14 @@ export default (config) => {
                     handleAPIError(request, response, error);
                 });
         },
+
+        async getSketchKmerArguments(request, response) {
```

Comment:
Might be less confusing to call this `getSpeciesConfig` - that seems to be the terminology elsewhere in the chain, and other config might get added in future.

Reply:
good catch i missed this in renaming haha